DXRMapFixups/DeusEx/Classes/DXRFixupM02.uc

```diff
@@ -231,6 +231,12 @@ function PreFirstEntryMapFixes()
         oot = Spawn(class'OnceOnlyTrigger');
         oot.Event='botordertriggerDoor';
         oot.Tag='botordertrigger';
+        SetAllLampsState(false, true, true);
```

Comment:
This feels really arbitrary, enabling certain combinations of lamp types per map.  I think this should be doing True, True, True  in DXRFixup's PreFirstEntry itself (so it applies to every map), and then if we really want exceptions, they could be individually set on a per-map basis.

Reply:
Nevermind, I see you're doing that

Reply:
I think what I've realized makes these exceptions feel bad is that at a glance, you can't really tell what the intended result is.  A comment on each exception documenting which lamps this is *intended* to turn off would improve them greatly (because it also makes it easier to catch unintended consequences)